components/common/Nav2.tsx

```diff
@@ -0,0 +1,89 @@
+import { Avatar, Input, Layout, Button, Badge } from 'antd';
+import { BellOutlined, MenuOutlined, UserOutlined } from '@ant-design/icons';
+import Search from "../search/search";
+import Link from 'next/link';
+import axios from "axios";
+import { SERVER_BASE_URL } from "../../lib/utils/constant";
+
+const { Header } = Layout;
+
+{/*async function isLoggedIn() {
+  const data = await axios.get(`${SERVER_BASE_URL}/user`);
+  return data;
+}*/}
+
+const Navbar = () => {
+
+  const headerStyle = {
```

Comment:
For all styles, use styled components instead, since that is what all the other files do as well.